src/main/webapp/app/entities/theory/theory.component.ts

```diff
@@ -1,22 +1,33 @@
+import { ActivatedRoute } from "@angular/router";
```

Comment:
What is `ActivatedRoute`?

Reply:
It's a class for receiving query params on page load

Reply:
Doesn't plain router fit our needs?

Reply:
Nope, it don't have necessary method for retrieving query params

Reply:
Ok